### PR TITLE
Update render-biblio.js

### PR DESCRIPTION
**Pull request by yash-agarwal17**
Fixed issue #2492. Biblio links have data link type = "biblio"

### CODE REVIEW

**Comment by yash-agarwal17** [comment]
@sidvishnoi  I am failing in running Karma Unit Tests. Also in my local machine i have an error "Error: Could not resolve '../../js/deps/builds/marked.js' from src/core/import-maps.js". Can you please guide me through this

**Comment by sidvishnoi** [comment]
> Also in my local machine i have an error "Error: Could not resolve '../../js/deps/builds/marked.js' from src/core/import-maps.js".

Did you run `npm install`?

File: src/core/render-biblio.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ function toRefContent(ref) {
 export function renderInlineCitation(ref) {
   const key = ref.replace(/^(!|\?)/, "");
   const href = `#bib-${key.toLowerCase()}`;
-  return hyperHTML`[<cite><a class="bibref" href="${href}">${key}</a></cite>]`;
+  return hyperHTML`[<cite><a class="biblio" href="${href}">${key}</a></cite>]`;
```
Comment (sidvishnoi) [inline src/core/render-biblio.js]: The task is to add a `data-link-type="biblio"`, not to change the class name.

**Comment by yash-agarwal17** [comment]
> > Also in my local machine i have an error "Error: Could not resolve '../../js/deps/builds/marked.js' from src/core/import-maps.js".
> 
> Did you run `npm install`?

yes after cloning the repo, i ran npm install.
Then on npm start, it is giving errors

File: src/core/render-biblio.js (hunk 1 above)
Comment (yash-agarwal17) [reply src/core/render-biblio.js]: > The task is to add a `data-link-type="biblio"`, not to change the class name.

okay! i'll try to fix that

**Comment by saschanaz** [comment]
> 
> 
> > > Also in my local machine i have an error "Error: Could not resolve '../../js/deps/builds/marked.js' from src/core/import-maps.js".
> > 
> > 
> > Did you run `npm install`?
> 
> yes after cloning the repo, i ran npm install.
> Then on npm start, it is giving errors

Could you do `npm run builddeps` and try again?

**Comment by yash-agarwal17** [comment]
> npm run builddeps

Thanks a lot, it worked

**Comment by sidvishnoi** [comment]
Fixed in https://github.com/w3c/respec/pull/2642

**Pull request closed without merging**